tests/scripts/all.sh

```diff
@@ -2285,6 +2285,37 @@ component_test_psa_crypto_config_accel_ffdh () {
 
     msg "test: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated FFDH"
     make test
+
+    msg "ssl-opt: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated FFDH alg"
+    tests/ssl-opt.sh -f "ffdh"
+}
+
+component_test_psa_crypto_config_reference_ffdh () {
+    msg "build: MBEDTLS_PSA_CRYPTO_CONFIG with accelerated FFDH"
+
+    # Start with full (USE_PSA and TLS 1.3)
+    scripts/config.py full
```

Comment:
Suggestion: use `helper_libtestdriver1_adjust_config "full"`, then most lines below become redundant (only need to keep the ones about `MBEDTLS_KEY_EXCHANGE_DHE_xxx_ENABLED` I think). This keeps the reference component closer to the accel component, too.